src/Entities/Player.cpp

```diff
@@ -193,13 +195,67 @@ bool cPlayer::Initialize(OwnedEntity a_Self, cWorld & a_World)
 
 	cPluginManager::Get()->CallHookSpawnedEntity(*GetWorld(), *this);
 
+	LOGINFO("Items size %d", this->KnownItems.size());
+	LOGINFO("Recipes size %d", this->KnownRecipes.size());
+	if (this->KnownRecipes.size() == 0)
```

Comment:
Don't compare size if all you're interested in is emptiness, use `empty()` instead.